pkg/vsphere/datastore/datastore.go

```diff
@@ -239,7 +239,7 @@ func (d *Helper) Mv(ctx context.Context, fromPath, toPath string) error {
 	from := path.Join(d.RootURL, fromPath)
 	to := path.Join(d.RootURL, toPath)
 	log.Infof("Moving %s to %s", from, to)
-	err := tasks.Wait(ctx, func(context.Context) (tasks.Waiter, error) {
+	err := tasks.Wait(ctx, func(context.Context) (tasks.ResultWaiter, error) {
```

Comment:
I had to for the tests. They were passing `waiter` objects I to the `wait` tests and we call `WaitForResult` in the wait call now. I thought about making the object.task change now. Wanted to clear this up first before potentially using the task object.